src/clj_kondo/impl/analyzer.clj

```diff
@@ -1642,6 +1643,14 @@
                         (analyze-like-let ctx expr)
                         [babashka.process $]
                         (babashka/analyze-$ ctx expr)
+                        ([re-frame.core reg-event-db]
+                         [re-frame.core reg-event-fx]
+                         [re-frame.core reg-event-ctx]
+                         [re-frame.core reg-sub]
+                         [re-frame.core reg-sub-raw]
+                         [re-frame.core reg-fx]
+                         [re-frame.core reg-cofx])
+                        (re-frame/analyze-reg ctx expr (symbol (str resolved-as-namespace) (str resolved-as-name)))
```

Comment:
Would it be useful if people can use `{:lint-as {foo.bar/baz re-frame.core/reg-event-fx}}`? In that case I think it's better to pass `resolved-namespace` instead of `resolved-as-namespace` (and also the `-name`)?

Reply:
@borkdude you mean change the arity to something like:
```suggestion
                        (re-frame/analyze-reg ctx expr resolved-namespace resolved-name)
```

Reply:
Oh, got it, done!

Reply:
I meant: use `resolved-namespace` instead of `resolved-as-namespace`. But this also seems fine.